core/src/main/java/fj/test/Arbitrary.java

```diff
@@ -733,6 +733,10 @@ public StringBuilder f(final String s) {
     return arbitrary(listOf(aa.gen));
   }
 
+  public static <A> Arbitrary<NonEmptyList<A>> arbNonEmptyList(final Arbitrary<A> aa) {
+    return arbitrary(Gen.listOf1(aa.gen).map(list -> NonEmptyList.fromList(list).some()));
+  }
+
```

Comment:
You're right, but I had to dig through the code to convince myself this is true.  The existing javadoc should be more specific.